packages/web/components/swap-tool/index.tsx

```diff
@@ -767,22 +769,23 @@ export const SwapTool: FunctionComponent<SwapToolProps> = observer(
                 ) : undefined}
                 {((swapState.quote?.tokenInFeeAmountFiatValue &&
                   swapState.quote?.swapFee) ||
-                  (swapState.networkFee && !swapState.isLoadingNetworkFee)) && (
-                  <div className="flex justify-between">
-                    <span className="caption">{t("swap.totalFee")}</span>
-                    <span className="caption text-osmoverse-200">
-                      {`≈ ${new PricePretty(
-                        DEFAULT_VS_CURRENCY,
-                        sum([
-                          swapState.quote?.tokenInFeeAmountFiatValue?.toDec() ??
-                            new Dec(0),
-                          swapState.networkFee?.gasUsdValueToPay?.toDec() ??
-                            new Dec(0),
-                        ])
-                      )} `}
-                    </span>
-                  </div>
-                )}
+                  (swapState.networkFee && !swapState.isLoadingNetworkFee)) &&
+                  featureFlags.swapToolSimulateFee && (
+                    <div className="flex justify-between">
+                      <span className="caption">{t("swap.totalFee")}</span>
+                      <span className="caption text-osmoverse-200">
+                        {`≈ ${new PricePretty(
+                          DEFAULT_VS_CURRENCY,
+                          sum([
+                            swapState.quote?.tokenInFeeAmountFiatValue?.toDec() ??
+                              new Dec(0),
+                            swapState.networkFee?.gasUsdValueToPay?.toDec() ??
+                              new Dec(0),
```

Comment:
Thoughts on exposing `totalFee` from swapState and doing that sum there?

I'd like to avoid the prev pattern of doing operations on data in the view layer.